cmd/parallel-manager.go

```diff
@@ -58,8 +58,8 @@ type ParallelManager struct {
 	// Current threads number
 	workersNum uint32
 
-	// Channel to receive tasks to run
-	queueCh chan task
+	// Channels to receive tasks to run
+	queueChannels []chan task
```

Comment:
This is always 2, correct?

Either just have 2 vars or `queueChannels [2]chan task`. I prefer the former, makes everything clearer and you don't have to pass around 0 and 1, but maybe a bool.